### ISSUE TITLE
Query tool - make submit button before data loads - recommendation user study #29

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#3210]
Make a submit button so users can make all filter selections before Query table loads.

Multiple users reported frustration with the loading time in the query data

accessibility issues - changes content without telling you that it is going to

**Comment by lpgoldstein** [comment]
This is an accessibility violation and will help site loading speed (m-23-22)

**Comment by lpgoldstein** [comment]
No longer relevant
